src/elm/clinical.ts

```diff
@@ -2,15 +2,17 @@ import { Expression } from './expression';
 import * as dt from '../datatypes/datatypes';
 import { Context } from '../runtime/context';
 import { build } from './builder';
+import ELM from '../types/elm';
 
 export class ValueSetDef extends Expression {
   name: string;
   id: string;
   version?: string;
 
-  constructor(json: any) {
+  constructor(json: ELM.ValueSetDef) {
+    // @ts-ignore
     super(json);
-    this.name = json.name;
+    this.name = json.name || '';
```

Comment:
After looking at the [logcial specification](https://cql.hl7.org/04-logicalspecification.html#valuesetdef) I actually think I defined the type of `name` incorrectly in this class. `name` is `0..1`, so I believe in the declaration of the class member, it should be `name?: string`. That should allow you to get rid of the `|| ''` assignment here. My bad!